server/services/event.ts

```
@@ -471,3 +479,49 @@ export async function getPointsOndeDoar(oldEvents: boolean = false) {
+      subscription: 1,
+    })
+    .lean();
+}
+
+// Get events starting in exactly 24 hours (with 1 hour window)
+export async function getEventsStartingIn24h() {
+  return getEventsStartingWithinHours(24, 1);
+}
+
+// Get events starting in exactly 2 hours (with 30 min window)
+export async function getEventsStartingIn2h() {
```

ESLint

[error] Async function 'getEventsStartingIn2h' has no 'await' expression.

(require-await)